docs/subsystems/user-docs.md

```diff
@@ -247,25 +247,37 @@ The source for macros is the Markdown files under
 multiple instances of particular content in the documentation, you can
 always create a new macro by adding a new file to that folder.
 
-### **Organization settings** `{!admin.md!}` macro
+### **Organization settings** `{!go-to-organization-settings.md!}` macro
 
-* **About:** Links to the **Organization settings** documentation. Usually
-preceded by a link to a particular section on the **Organization settings**
+* **About:** Describes steps to get to **Organization settings**. Usually
+followed by a reference to a particular section on the **Organization settings**
 page.
 
 * **Contents:**
     ```md
-    tab of the [Organization settings](/help/edit-administrator-settings) page.
+    1. From your desktop, click on the **gear**
+       (<i class="icon-vector-cog"></i>) in the upper right corner.
+
+    1. Select **Settings**.
+
+    1. Select **Organization** from the top left corner of the settings screen.
     ```
 
 * **Example usage and rendering:**
```

Comment:
I would replace this section with a link to an article that uses it. (And if it's used in 2 different ways, a link to both ways it is used.)
Can also be a separate commit from the big one.